docs/UserGuide.md

```diff
@@ -128,6 +128,106 @@ Command Options:
 > Attribute “amount” must be a number (received: 5.80.)
 > ```
 
+
+### Delete Expense
+Allows a user to delete a previously added expense and all information associated with that expense.
+
+Command: `delete <name>`
+
+Command Argument: `name` represents the exact name of the expense intended to be deleted. 
+Has to exactly match a given expense, or else the command will do nothing, 
+so as to ensure the integrity of user data.
+
+#### Expected Outputs
+
+##### Successful Execution
+
+###### Example 1
+
+>**Case**:  Delete expense named “friday mcdonalds”
+>
+>**Input**: `delete "friday mcdonalds"`
+>
+>**Output**: Successfully deleted expense “friday mcdonalds”!
```

Comment:
Shall we standardize all outputs to be wrapped in the fenced code block? I saw that @elhy1999 did it so I followed